Core/MIPS/MIPSDebugInterface.cpp

```diff
@@ -162,17 +162,20 @@ class MipsExpressionFunctions: public IExpressionFunctions
 	bool getMemoryValue(uint32_t address, int size, uint32_t& dest, char* error) override {
 		// We allow, but ignore, bad access.
 		// If we didn't, log/condition statements that reference registers couldn't be configured.
-		bool valid = Memory::IsValidRange(address, size);
+		uint32_t valid = Memory::ValidSize(address, size);
+		uint8_t buf[4]{};
+		if (valid != 0)
+			memcpy(buf, Memory::GetPointerUnchecked(address), valid);
 
 		switch (size) {
 		case 1:
-			dest = valid ? Memory::Read_U8(address) : 0;
+			dest = (buf[3] << 24) | (buf[2] << 16) | (buf[1] << 8) | buf[0];
```

Comment:
Hm, how does this do the same as the old code? previously it returned maximum an 8-bit value. I think this case is swapped with "case 4".

Reply:
Oops, thanks.  I guess I only tried something where the lowest byte mattered anyway...

-[Unknown]